Index.html

```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="works">
+        <button class="htmlButton buttons"></button>
+        <button class="cssButton buttons"></button>
+        <button class="jsButton buttons"></button>
+      </div>
+      <button class="seeProject">See project</button>
+    </div>
+    <div class="space"></div>
+    <div class="space"></div>
+    <div class="lastCard">
+      <div class="snapshotFourth"></div>
+      <h3 class="stories">Multi-Post Stories</h3>
+      <ul class="canopyList">
+        <li class="canopy">CANOPY</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="backEnd">Back End Dev</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="works">
+        <button class="htmlButton buttons"></button>
+        <button class="cssButton buttons"></button>
+        <button class="jsButton buttons"></button>
+      </div>
+      <button class="seeProject">See project</button>
+    </div>
+    <div class="space"></div>
+  </section>
+  <section class="aboutMe">
+    <div class="background">
+      <h1 class="introTitle introTwo">About Myself</h1>
+      <p class="textIntro">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <h2 class="connect">LET'S CONNECT</h2>
+      <ul class="icons subIcons">
+        <li class="gitHub"><img src="Assests/Vectorgithub.svg"></li>
+        <li><img src="Assests/Linkedin iconlinked.svg"></li>
+        <li><img src="Assests/Vectorangerllist.svg"></li>
+        <li class="twitter bird"><img src="Assests/Vectortwitter.svg"></li> 
+        <li class="medium"><img src="Assests/Vectormedium.svg"></li> 
+      </ul>
+      <button class="secondButton">Get my resume</button>
+      <ul class="abilities">
+        <li class="languages"><aside class="textOne">Languages</aside><img class="droplet down" src="Assests/preview.png">
+          <ul class="subLanguages">
+            <li class="iconw"><img class="image" src="Assests/Ellipse 1jsLogo.svg">JavaScript</li>
+            <li class="iconw"><img class="image" src="Assests/Ellipse 1htmlLogo.svg">HTML</li>
+            <li class="iconw"><img class="image" src="Assests/Ellipse 1css.svg">CSS</li>
+          </ul>
+        </li>
+        <li class="frameworks languages">Frameworks<img class="droplet side" src="Assests/droplet.png"></li>
+        <li class="skills languages">Skills<img class="droplet sideTwo" src="Assests/droplet.png"></li>
+      </ul>
+    </div>
```

Comment:
- [ ] Nice efforts here 👏 . However, you have a better design that will give a nice user experience and also stick to the to design, kindly take note of the paddings, and margins provided on the original design 👍 
- For the button, you can see the height and width of yours is a bit adjusted. do make sure it has same dimensions as te design given 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50990925/235335162-bebd4b0c-71d9-4c13-97f2-77ce34c354bc.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/235335175-dafd13b8-cda1-473d-9b2e-547a007b57d5.png)